server/src/models/run.js

```diff
@@ -727,6 +729,18 @@ module.exports = {
 		if (queryParams.order) {
 			if (queryParams.order === 'date')
 				queryOptions.order = [['createdAt', 'DESC']];
+			if (queryParams.order === 'time')
+				queryOptions.order = [[sequelize.literal('ticks * tickRate'), 'ASC']];
+		}
+		if (queryParams.mapName) {
+			queryOptions.include.push({
+				model: Map,
+				as: 'map',
+				where: { name: {[Op.startsWith]: queryParams.mapName} }
```

Comment:
We will want to use name matching instead of `startsWith`. See [how the map model does it here](https://github.com/momentum-mod/website/blob/646f8ab8f3eeb6d85d333c2b8266165166b6c4df/server/src/models/map.js#L171).

Reply:
Fixed in 2f43ccd. Let me know if it still doesn't look right.